tests/install-branch.js

```diff
@@ -0,0 +1,23 @@
+const puppeteerutils = require('./puppeteer-utils');
+
+const setup = async () => {
+  let {page, browser} = await puppeteerutils.login();
```

Comment:
Would it make sense to move this into the below try/catch block too?

Reply:
And then we don't need a try/catch block in lines 17-22, because the setup function already has a try/catch block around all of it.

Reply:
The problem is that I want `browser` to be defined so I can close it in the `finally` block